jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskdetails/TaskDetailsPresenter.java

```diff
@@ -98,6 +100,14 @@ protected void setTaskDetails(String status,
         view.setUser(actualOwner);
         view.setTaskStatus(status);
         view.setTaskPriority(priority);
+        if (processInstanceId == null) {
+            view.setProcessInstanceId("None");
```

Comment:
@nmirasch perhaps hide these fields in the UI if null?

Reply:
@nmirasch @cristianonicolai I wonder if this this is the right place to even consider tasks without Porcess Instance ID or Process ID. Taking into account that starting from v7 there can be no tasks outside the process context, if those tasks even appear they apparently represent an illegal state and a bug on Kie Server side.
I think this validation should be done somewhere in backend code instead. Not sure what are the options there, though, as if we are just taking the result set from Kie Server and sending it directly to the task list without any validation, this is then not possible.
What I'm trying to say is that this task shouldn't even be listed to begin with, hardly to get to the point of looking at its details in UI.